pkg/containers/containers.go

```diff
@@ -50,14 +53,28 @@ func InitContainers() (*Containers, error) {
 		}
 	}
 
+	runtimeService := RuntimeInfoService()
```

Comment:
The only reason I did this was because maps aren't automatically allocated on struct construction, so I had to include some constructor. I don't think we should call this function New due to the package scope but we can prefix this function with a `New`, so it will be `NewRuntimeInfoService` instead.

I do think that what we have now expresses the intent clearly enough though.

Reply:
Have you thought about this one ?

Reply:
Are you talking about the name or where we're registering?
On the former, i'm still of the same opinion that it shouldn't be New(), I'm open to other suggestions (for the struct as well as the corresponding constructor).
On the latter, I'm not really sure where else we would register them, initially all the enrichers were hardcoded in the service and a switch was used to derive which one of them to use. So really the only other way to register that I see would be even more inwards. Externally from containers, I'm not sure where it would be appropriate to do, since the containers package to me seems to be the root of this feature, initializing it there made sense to me.

Reply:
I've refactored the registration process by having the service receive the `Sockets` as an arguments before. WDYT?